sphinx.testing.path has been removed from Sphinx 9+, affects conftest.py

```
+ /usr/bin/pytest -v
ImportError while loading conftest '/builddir/build/BUILD/python-sphinx-math-dollar-1.2.1-build/sphinx-math-dollar-1.2.1/conftest.py'.
conftest.py:2: in <module>
    from sphinx.testing.path import path
E   ModuleNotFoundError: No module named 'sphinx.testing.path'
```

sphinx.testing.path has been removed from Sphinx 9+

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -21,11 +21,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v5
+      - uses: actions/checkout@v6
       - name: Set up Python
         uses: actions/setup-python@v6
         with:
-          python-version: '3.12'
+          python-version: '3.14'
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip

--- a/.github/workflows/pythonpackage.yml
+++ b/.github/workflows/pythonpackage.yml
@@ -8,11 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', 'pypy-3.8',
-                         'pypy-3.9', 'pypy-3.10']
+        python-version: ['3.11', '3.12', '3.13', '3.14', 'pypy-3.11']
 
     steps:
-    - uses: actions/checkout@v5
+    - uses: actions/checkout@v6
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v6
       with:

--- a/conftest.py
+++ b/conftest.py
@@ -1,5 +1,6 @@
+from pathlib import Path
+
 import pytest
-from sphinx.testing.path import path
 
 
 try:
@@ -14,4 +15,4 @@ pytest_plugins = 'sphinx.testing.fixtures'
 
 @pytest.fixture(scope='session')
 def rootdir():
-    return path(__file__).parent.abspath() / 'sphinx_math_dollar' / 'tests'
+    return Path(__file__).parent.resolve() / 'sphinx_math_dollar' / 'tests'

--- a/setup.py
+++ b/setup.py
@@ -21,10 +21,14 @@ setuptools.setup(
     packages=setuptools.find_packages(include=["sphinx_math_dollar"]),
     classifiers=[
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
+        "Programming Language :: Python :: 3.14",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.11",
     install_requires=["sphinx"],
     license="MIT",
 )

